docs/mkdocs.yml

```diff
@@ -0,0 +1,143 @@
+site_url: ""
+
+repo_url: https://github.com/JuliaData/DataFrames.jl
+repo_name: DataFrames.jl
+edit_uri: "" # No edit button, as some of our pages are in /docs and some in /examples via symlink, so it's impossible for them all to be accurate
+
+strict: true # Don't allow warnings during the build process
+extra_javascript:
+  # The below three make MathJax work, see https://squidfunk.github.io/mkdocs-material/reference/mathjax/
+  - _static/mathjax.js
+  - https://polyfill.io/v3/polyfill.min.js?features=es6
+  - https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js
+
+extra:
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/JuliaData/DataFrames.jl
+    - icon: fontawesome/brands/twitter
+      link: https://twitter.com/BogumilKaminski
+
+extra_css:
+  - stylesheets/custom.css
+  - assets/Documenter.css
+
+extra_javascript:
+  - javascripts/mathjax.js
+  - https://polyfill.io/v3/polyfill.min.js?features=es6
+  - https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js
+
+# TODO
+# https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/
+#extra:
+#  version:
+#    provider: mike
+
+markdown_extensions:
+  - tables
+  - admonition
+  - toc:
+      permalink: "¤" # Adds a clickable permalink to each section heading
+      toc_depth: 4
+  - pymdownx.arithmatex: # Render LaTeX via MathJax
+      generic: true
+  - pymdownx.details # Allowing hidden expandable regions denoted by ???
+  - pymdownx.highlight
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.superfences # Seems to enable syntax highlighting when used with the Material theme.
+  - pymdownx.tasklist:
+      custom_checkbox: true
+  - def_list
+  - pymdownx.tabbed:
+      alternate_style: true 
+  - attr_list
+  - md_in_html
+plugins:
+  - search # default search plugin; needs manually re-enabling when using any other plugins
+  - autorefs # Cross-links to headings
+  - include_exclude_files:
+      exclude:
+#        - "_overrides"
+#  - mknotebooks # Jupyter notebooks
+#  - mkdocs-video
+nav:
+  - "Home": "index.md"
+  - "First Steps with DataFrames": "./man/basics.md"
+  - "User Guide" :
+    - "Getting Started": "./man/getting_started.md"
+    - "Working with DataFrames": "./man/working_with_dataframes.md"
+    - "Importing and Exporting Data (I/O)": "./man/importing_and_exporting.md"
+    - "Joins": "./man/joins.md"
+    - "Split-apply-combine": "./man/split_apply_combine.md"
+    - "Reshaping": "./man/reshaping_and_pivoting.md"
+    - "Sorting": "./man/sorting.md"
+    - "Categorical Data": "./man/categorical.md"
+    - "Missing Data": "./man/missing.md"
+    - "Data manipulation frameworks": "./man/querying_frameworks.md"
+    - "Comparison with Python/R/Stata": "./man/comparisons.md"
+  - "API" :
+    - "Types": "./lib/types.md"
+    - "Functions": "./lib/functions.md"
+    - "Indexing": "./lib/indexing.md"
+    - "Metadata": "./lib/metadata.md"
+#    hide("Internals" => "lib/internals.md")
```

Comment:
why `hide` is not needed any more? I would assume that Documenter.jl would error if we do not cover this file in docs with `hide`.

Reply:
as it is, it just outputs a warning saying that this is file is available but not in the menu bar.

Reply:
```
lalonso@pc DataFrames.jl % cd docs 
lalonso@pc docs % mkdocs serve
INFO     -  Building documentation...
INFO     -  Cleaning site directory
INFO     -  The following pages exist in the docs directory, but are not included in the "nav"
            configuration:
              - assets/README.md
              - lib/internals.md
INFO     -  Documentation built in 2.54 seconds
INFO     -  [13:44:20] Watching paths for changes: 'docs', 'mkdocs.yml'
INFO     -  [13:44:20] Serving on http://127.0.0.1:8000/
```